apps/balancer-tools/src/components/Tabs.tsx

```diff
@@ -78,16 +78,19 @@ function TabItemTrigger({
   tabName,
   color,
   onClick,
+  triggerBgColor = "bg-blue3",
```

Comment:
why not simply `triggerClassNames`? Also not sure I love this pattern tbh... would rather we have the composable pattern we started introducing in the app (and also what [shadcn](https://ui.shadcn.com/docs/components/tabs) does)